Description: Pod::Usage changed its output in 1.65:
 https://metacpan.org/diff/file?target=MAREKR%2FPod-Usage-1.65%2F&source=MAREKR%2FPod-Usage-1.64%2F#lib/Pod/Usage.pm
 Update t/tests.t to handle both versions.
Origin: vendor
Bug-Debian: https://bugs.debian.org/825428
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-06-05

--- a/t/tests.t
+++ b/t/tests.t
@@ -48,11 +48,17 @@
 
 # Usage, invalid arguments.
 
-my $usage =
+use Pod::Usage;
+my $usage = ( $Pod::Usage::VERSION < 1.65 ?
         'Usage:
     blhc [*options*] *<dpkg-buildpackage build log file>..*
 
-';
+'
+    :
+        'Usage:
+    blhc [options] <dpkg-buildpackage build log file>..
+
+');
 is_blhc '', '--invalid', 2,
         "Unknown option: invalid\n"
         . $usage;
@@ -79,7 +85,7 @@
 
 is_blhc '', '--help', 1,
         qr/^Usage:
-    blhc \[\*options\*\] \*<dpkg-buildpackage build log file>\.\.\*
+    blhc \[\*?options\*?\] \*?<dpkg-buildpackage build log file>\.\.\*?
 
 Options:
 /s;
@@ -118,23 +124,14 @@
 
 # Invalid option.
 is_blhc 'ignore-flag', '--ignore-arch-flag -g', 2,
-        'Value "-g" invalid for option ignore-arch-flag ("arch:flag" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value "-g" invalid for option ignore-arch-flag ("arch:flag" expected)'
+        . "\n$usage";
 is_blhc 'ignore-flag', '--ignore-arch-flag -g:', 2,
-        'Value "-g:" invalid for option ignore-arch-flag ("arch:flag" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value "-g:" invalid for option ignore-arch-flag ("arch:flag" expected)'
+        . "\n$usage";
 is_blhc 'ignore-flag', '--ignore-arch-flag :amd64', 2,
-        'Value ":amd64" invalid for option ignore-arch-flag ("arch:flag" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value ":amd64" invalid for option ignore-arch-flag ("arch:flag" expected)'
+        . "\n$usage";
 
 # Wrong architecture.
 is_blhc 'ignore-flag', '--ignore-arch-flag amd64:-g', 8,
@@ -181,23 +178,14 @@
 
 # Invalid option.
 is_blhc 'ignore-line', '--ignore-arch-line .+', 2,
-        'Value ".+" invalid for option ignore-arch-line ("arch:line" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value ".+" invalid for option ignore-arch-line ("arch:line" expected)'
+        . "\n$usage";
 is_blhc 'ignore-line', '--ignore-arch-line .+:', 2,
-        'Value ".+:" invalid for option ignore-arch-line ("arch:line" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value ".+:" invalid for option ignore-arch-line ("arch:line" expected)'
+        . "\n$usage";
 is_blhc 'ignore-line', '--ignore-arch-line :amd64', 2,
-        'Value ":amd64" invalid for option ignore-arch-line ("arch:line" expected)
-Usage:
-    blhc [*options*] *<dpkg-buildpackage build log file>..*
-
-';
+        'Value ":amd64" invalid for option ignore-arch-line ("arch:line" expected)'
+        . "\n$usage";
 
 # Wrong architecture.
 is_blhc 'ignore-line', '--ignore-arch-line "amd64:.+"', 8,
